### PR TITLE
sql: add several system information functions

### PR DESCRIPTION
**Pull request by benesch**
See individual commits for details!

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/6110)

### CODE REVIEW

**Comment by philip-stoev** [comment]
Can I add a few quick tests here, just to make sure that the functions are call-able and return non-null, non-zero values?

File: src/pgrepr/src/oid.rs [hunk 1]
```diff
@@ -60,3 +60,6 @@ pub const OP_MOD_F32_OID: u32 = 16_431;
 pub const OP_MOD_F64_OID: u32 = 16_432;
 pub const OP_UNARY_PLUS_OID: u32 = 16_433;
 pub const FUNC_MZ_SLEEP_OID: u32 = 16_434;
+pub const FUNC_MZ_SESSION_ID_OID: u32 = 16_435;
+pub const FUNC_MZ_UPTIME_OID: u32 = 16436;
```
Comment (sploiselle) [inline src/pgrepr/src/oid.rs:64]: immaterial but the rest of these are underscore-comma'ed for legibility

**Review by sploiselle — APPROVED** [review]

**Comment by benesch** [comment]
@philip-stoev that'd be great!

**Review by philip-stoev — APPROVED** [review]
Thank you, pushed tests.

**Pull request merged**